west.yml

```diff
@@ -86,7 +86,7 @@ manifest:
       revision: 30b7efa827b04d2e47840716b0372737fe7d6c92
       path: tools/net-tools
     - name: hal_nxp
-      revision: 62b46313b8e07e2d5062fd62f14ee9dfeb201b76
+      revision: pull/29/head
```

Comment:
@galak 
This should not have been merged as is

Reply:
would be nice to have check against pull reference committing in the CI.

Reply:
Has been fixed afterwards it seems.